### DISCUSSION TITLE
kubeadmiral can schedule no native k8s crd, like argo workflow or volcano job?

### DISCUSSION THREAD

**Discussion by haiker2011** [kubewharf/kubeadmiral#140] [Q&A] [answered]
I find kubeadmiral transfer native k8s api to fedrate type, now support  no native k8s crd, like argo workflow or volcano job?

**Comment by SOF3** [comment, marked as answer]
KubeAdmiral supports arbitrary object types through creating the FederatedTypeConfig. In addition to the default ones [here](https://github.com/kubewharf/kubeadmiral/blob/fef9f08d093cf4a7e491bf28865bb3cf0e593c9d/config/sample/host/01-ftc.yaml), you can also create new federated types by applying a FederatedTypeConfig with the corresponding type information.

**Comment by gary-lgy** [comment]
In addition to what @SOF3 said, we are in the process of improving the user experience. In the future we will be providing guides for common operations like enabling the federation of a new resource type. We are also considering automating some operations with a CLI for an even smoother experience. :)

**Comment by haiker2011** [comment]
That's sounds great!
